source/features/hidden-review-comments-indicator.css

```diff
@@ -29,7 +22,7 @@
 
 .rgh-comments-indicator button {
 	position: absolute;
-	right: 95%;
+	left: -40px;
```

Comment:
`left: -40px` only works for the specific icon size, font size and character count.
`right: 100%` is the correct alignment and it works with any amount of text in this button.

The only problem lies in the button’s content itself, which seems to have excessive padding (which exists to expand its clickable area, but it should be counteracted by negative margins)